examples/multiBarChart.html

```diff
@@ -33,6 +33,7 @@
     var test_data = stream_layers(3,10+Math.random()*100,.1).map(function(data, i) {
         return {
             key: 'Stream' + i,
+            stacked: true,
```

Comment:
Your use of this attribute isn't backward compatible... by default it should stack like it has always done, and the new attribute should make the behavior change - i.e. that it should be un-stackable.